src/Phpro/SoapClient/BridgeBundle/Type/SoapCall.php

```diff
@@ -133,11 +142,16 @@ public function getRequestHeaders()
      */
     public function getResponseBody()
     {
-        $response = new \DOMDocument();
-        $response->preserveWhiteSpace = false;
-        $response->formatOutput = true;
-        $response->loadXML($this->responseBody);
-        return $response->saveXML();
+        if ($this->responseBody) {
```

Comment:
Can you invert the if and add an early return instead?
Is this actually an error, or should the message say something like 'Empty response'?